src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -322,5 +323,25 @@ internal static bool IsMsOrElastic(byte[] array)
 
 			return true;
 		}
+
+		protected string ParseCaptureBody(ConfigurationKeyValue kv)
+		{
+			var captureBodyInConfig = kv.Value;
+			if (string.IsNullOrEmpty(captureBodyInConfig))
+			{
+				return ConfigConsts.SupportedValues.CaptureBodyOff;
```

Comment:
nit: why not `return DefaultValues.captureBody` here, as below?

Reply:
In order not have to deal with a null/empty value passed in the config file or env variables. Let me know if you prefer to change that. I'd feel better to align the code to the preference of the devs with more experience in this project.

Reply:
I'm not sure I can follow you.

So, this is the part I talk about:

```
if (string.IsNullOrEmpty(captureBodyInConfig))
{
   return ConfigConsts.SupportedValues.CaptureBodyOff;
}
```

All I suggest is to replace `ConfigConsts.SupportedValues.CaptureBodyOff` with `DefaultValues.captureBody`, because in case the entered value is `null` or an empty string we want to return the default value - which is indeed the same as `ConfigConsts.SupportedValues.CaptureBodyOff`, it just communicates the intent better.

So in this sense... we do deal with null/empty values: we simply return the default in that case.

Reply:
OK. Will do.

Reply:
Done. code is pushed.